tests/performance/lookup/query.cpp

```diff
@@ -0,0 +1,27 @@
+#include <cstdlib>
+#include <cstdio>
+#include <cstdint>
+#include <sstream>
+
+void
+query(const char * field, unsigned int keys_per_query, unsigned long upper_limit) {
+    std::ostringstream keys;
+    keys << "%22" << (rand()%upper_limit) << "%22%3A1";
+    for (unsigned int i(1); i < keys_per_query; i++) {
+        keys << ",%22" << (rand()%upper_limit) << "%22%3A1";
+    }
+    const char * queryPrefix = "/search/?yql=select%20*%20from%20sources%20*%20where%20weightedSet";
+    printf("%s(%s,%7B%s%7D)%3B\n", queryPrefix, field, keys.str().c_str());
+}
+
+int
+main(int argc, char **argv) {
+    long numQueries = atoi(argv[1]);
```

Comment:
Consider calling srand() to produce same random data between runs.

Reply:
Done. Note that the seed is 1 if srand is never called.